operators/src/upgrade/common/constants.rs

```diff
@@ -0,0 +1,36 @@
+
+#[macro_export]
+macro_rules! upgrade_group {
+    () => {
+        "openebs.io"
+    };
+    ($s:literal) => {
+        concat!(upgrade_group!(), "/", $s)
+    };
+}
+
+#[macro_export]
+macro_rules! upgrade_labels {
+    ($s:expr) => {
+        btreemap! {
+            APP => $s,
+            LABEL => $s,
+        }
+        .iter()
+        .map(|(k, v)| (k.to_string(), v.to_string()))
+        .collect()
+    };
+}
+
+pub const APP: &str = "app.kubernetes.io/component";
```

Comment:
Why are all these pub, do we need to expose these to an api?

Reply:
No, pub is not required. Will update it.